Help/cobot_help.rst

```diff
@@ -0,0 +1,122 @@
+Usage of cobot
+==============
+
+.. note::
+
+Display all of the help commands that Hubot knows about
+---------------------------------------------------------
+
+cobot help
+
+::
+
+    cobot help
+
+Display all help comments that match <query>
+----------------------------------------------
+
+cobot help <query>
+
+::
+
+    cobot help explain
+
+Assigns issus <issus_link> to user
+------------------------------------
+
+conbot assign <issus_link>
+
+::
+
+    cobot assign https://github.com/coala/coala/issues/3845
+
+Unassign the user from the issus <issue_link>
+------------------------------------------------
+
+cobot unassign <issus_link>
+
+::
+
+    cobot unssign https://github.com/coala/coala/issues/3845
+
+Create a Github issus given the title and repository
+-------------------------------------------------------
+
+cobot new issus
```

Comment:
typo, also mention about the alias `file` and the repository parameter for which the issue is filed,

 `cobot new issue coala-repository-name`